code_search/ks-web-app/app.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: 0.2.0
+environments:
+  cs_demo:
+    destination:
+      namespace: cs-web-app
+      server: https://35.185.115.154
```

Comment:
i see similar IP in a few places? was this intended? to me https://kubernetes.default always works.

Reply:
kubernetes.default should only work if you are running in cluster.

But we want our ksonnet to be able to work off cluster. In which case we need to specify the actual IP of the master.